pages.es/linux/grub-mkconfig.md

```diff
@@ -0,0 +1,16 @@
+# grub-mkconfig
+
+> Generar un archivo de configuracion de GRUB.
+> Para mas informacion: <https://www.gnu.org/software/grub/manual/grub/html_node/Invoking-grub_002dmkconfig.html>.
```

Comment:
We have translations for [more information](https://github.com/tldr-pages/tldr/blob/6d01f29bbd9135553d255574197f293d2942885a/scripts/set-more-info-link.py#L18).
```suggestion
> Más información: <https://www.gnu.org/software/grub/manual/grub/html_node/Invoking-grub_002dmkconfig.html>.
```